src/cf-base.less

```diff
@@ -536,26 +506,20 @@ table {
 
 th,
 td {
-    padding: unit(12px / @base-font-size-px, em) unit(15px / @base-font-size-px, em);
-    background: @td-bg;
+    padding: 10px;
 
     thead & {
         color: @thead-text;
         background: @thead-bg;
-    }
-
-    tbody > tr:nth-child(odd) > & {
-        background: @td-bg-alt;
-    }
-
-    .compact-table & {
-        padding: unit(7px / @base-font-size-px, em) unit(10px / @base-font-size-px, em);
+        font-size: .875em;
+        text-transform: uppercase;
+        .webfont-demi();
```

Comment:
I'd ask the designers if it _should_ match the h5 specifically, because from the work they've been doing to reduce different type settings, it seems it should.